src/components/ItemsForSale.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react'
```

Comment:
I'm slowly trying to create consistency across classes. The preference is now
1) Global imports
2) Local imports
3) CSS imports